frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/utils.ts

```diff
@@ -1,9 +1,27 @@
 import * as _ from 'lodash';
-import { TektonParam } from '../../../types';
+import { getResourceModelFromTaskKind } from '../../../utils/pipeline-augment';
+import { PipelineKind, PipelineTask, TektonParam } from '../../../types';
+import { ResourceModelLink } from '../resource-overview/DynamicResourceLinkList';
 
 export const removeEmptyDefaultFromPipelineParams = (parameters: TektonParam[]): TektonParam[] =>
   _.map(
     parameters,
     (parameter) =>
       _.omit(parameter, _.isEmpty(parameter.default) ? ['default'] : []) as TektonParam,
   );
+
+export const getPipelineTaskLinks = (pipeline: PipelineKind) => {
+  const tasks = [pipeline.spec.tasks, pipeline.spec.finally];
+  return tasks.reduce((taskLinks: ResourceModelLink[][], currTasks) => {
```

Comment:
`[] => []` of the same length is a `.map`. Please review my [slides from the workshop](https://coreos.slack.com/archives/CHGU4P8UU/p1616086865027400?thread_ts=1613727984.359100&cid=CHGU4P8UU).